src/modules/mc_pos_control/mc_sequencer.cpp

```diff
@@ -0,0 +1,293 @@
+#include <px4_config.h>
+
+#include "mc_sequencer.h"
+
+#define SEQ_ENTRY(type, thrust, delay, r, p, y) (seq_entry_s {Seq_state::type, thrust, {r,p,y}, delay})
+
+sequence *get_sequence(sequence_set entry)
+{
+	struct sequence *result = nullptr;
+
+	switch (entry) {
+	case coord_turn:
+		result = (sequence *) new sequence(3);
+		result->entries[0] = SEQ_ENTRY(RATE, 0.6f, 1.0f, 0.7f, -0.25f, 0.4f);
+		result->entries[1] = SEQ_ENTRY(RATE, 0.7f, 26.0f, 0.0f, 0.0f, 0.73f);
+		result->entries[2] = SEQ_ENTRY(ATTITUDE, 0.5f, 0.0f, 0.0f, 0.0f, NAN);
+		break;
+
+#if defined (CONFIG_ARCH_BOARD_PX4FMU_V4) || defined (CONFIG_ARCH_BOARD_SITL)
```

Comment:
yes, I'd like to limit this feature to boards with > 1MB of flash, but not sure if that's possible